### DISCUSSION TITLE
How to make `diff` options works on verbose output?

### DISCUSSION THREAD

**Discussion by latipun7** [twpayne/chezmoi#1994] [Q&A] [answered]
I set custom diff command in my `.chezmoi.yaml.tmpl`:
```yaml
diff:
  pager: delta
  command: delta
```

It works when the diff is invoked thru `chezmoi diff` or when there's conflict and select `diff` option.

But, I'd like to see custom diff in verbose output too. For example when I run `chezmoi apply -v`, the output currently use built-in git diff format like this:
```shell
❯ chezmoi apply -v ~/.config/git/config
diff --git a/.config/git/config b/.config/git/config
index 7e2fdf2cae5e269c70f781e2b7a157a5133e9117..16319679de505c9e0b8c4c32a6d35b3773312c2a 100600
--- a/.config/git/config
+++ b/.config/git/config
@@ -35,8 +35,6 @@   features = side-by-side line-numbers decorations navigate hyperlinks
   syntax-theme = catppuccin
-  plus-style = syntax "#003800"
-  minus-style = syntax "#3f0001"
 
 [delta "decorations"]
   file-style = bold yellow ul
```
I would like to see the diff with custom diff and the diff pager disabled in verbose output, how can I do that?
If the current state is not available, please make it new features.
Thank you.

**Comment by twpayne** [comment, marked as answer]
Thanks for opening this. Indeed, the current state is the `chezmoi apply` always uses the builtin diff rather than the user's preferred diff command. I'll convert this into an issue to track it as a feature request.

**Reply by latipun7** [reply]
Ok thank you for your follow up. I'll mark this as answered since a new issue has already been created. Thanks.
